css/_uswds-theme-custom-styles.scss

```diff
@@ -519,6 +519,19 @@ img {
   }
 }
 
+.site-figure {
+  @include u-display("flex");
+  @include u-flex("column");
+  @include u-flex("align-center");
+  @include u-margin(0);
+}
+
+.site-figcaption {
+  @include u-font-size("lang", "2xs");
+  @include u-margin-y(2);
+  color: color("gray-60");
```

Comment:
Color is set via `color: color($site-ink);`. 

Alternatively, we can remove and it should inherit the correct color.

Reply:
Thanks! Fixed in [ba6df02](https://github.com/uswds/uswds-site/pull/1751/commits/ba6df02ca7f5de9e0b88c84dc1cb34aac64746ad)